kubespawner/spawner.py

```diff
@@ -433,6 +442,15 @@ def _hub_connect_port_default(self):
         """
     )
 
+    user_storage_match_labels = Dict(
+        help="""
```

Comment:
Ah, so is it for the case where the PV is not dynamically created by using a StorageClass?